packages/devtools_app/lib/src/shared/http/_http_date.dart

```diff
@@ -89,187 +89,6 @@ class HttpDate {
     return sb.toString();
   }
 
-  /**
-   * Parse a date string in either of the formats
-   * [RFC-1123](http://tools.ietf.org/html/rfc1123 "RFC-1123"),
-   * [RFC-850](http://tools.ietf.org/html/rfc850 "RFC-850") or
-   * ANSI C's asctime() format. These formats are listed here.
-   *
-   *     Thu, 1 Jan 1970 00:00:00 GMT
-   *     Thursday, 1-Jan-1970 00:00:00 GMT
-   *     Thu Jan  1 00:00:00 1970
-   *
-   * For more information see [RFC-2616 section
-   * 3.1.1](http://tools.ietf.org/html/rfc2616#section-3.3.1
-   * "RFC-2616 section 3.1.1").
-   */
-  static DateTime parse(String date) {
```

Comment:
@CoderDake @bkonyi is this a bug that this is unused?

Reply:
I can't seem to find any uses of it anywhere.
Something about it seems like it was copied from somewhere anyways, especially if we don't have tests for it. As such it would probably be brittle enough that it would be ok to let go of.